### ISSUE TITLE
outdated

### ISSUE THREAD

**Issue by pazport** [qdm12/teamspeak-server-alpine#5]
hey mate. any chance of updating this please. when i look at logs. i see this 

2021-02-01 22:31:41.944722|CRITICAL|Accounting    |   |The default license has expired. Please use the latest server version.,

**Comment by qdm12** [comment]
Thanks for taking the time to go the extra mile and create an issue!

It should be fixed in 5c7e5c40ee3c1764805ece8002ba4ba2541889d5 you can try by re-pulling `:latest`.

**Comment by pazport** [comment]
hey mate. sorry for the very late reply. i did the latest pull 2 days. and it keeps throwing me this error 

``2021-05-19 00:31:24.537197|CRITICAL|Accounting    |   |The default license has expired. Please use the latest server version.,
2021-05-19 00:31:24.537124|WARNING |Accounting    |   |Unable to open licensekey.dat, falling back to limited functionality``

**Comment by qdm12** [comment]
Should be fixed now (wait for the [build](https://github.com/qdm12/teamspeak-server-alpine/runs/2615950317))

**Comment by pazport** [comment]
hey mate. sorry for the long reply. i notced other day it stopped working again. and i am getting the same error all over 

```
2021-07-24 14:58:38.168316|INFO    |ServerLibPriv |   |TeamSpeak 3 Server 3.12.1 (2020-03-27 10:38:47)
2021-07-24 14:58:38.168586|INFO    |ServerLibPriv |   |SystemInformation: Linux 4.15.0-151-generic #157-Ubuntu SMP Fri Jul 9 23:07:57 UTC 2021 x86_64 Binary: 64bit
2021-07-24 14:58:38.168630|INFO    |ServerLibPriv |   |Using hardware aes
2021-07-24 14:58:38.168918|INFO    |DatabaseQuery |   |dbPlugin name:    SQLite3 plugin, Version 3, (c)TeamSpeak Systems GmbH
2021-07-24 14:58:38.168983|INFO    |DatabaseQuery |   |dbPlugin version: 3.11.1
2021-07-24 14:58:38.169232|INFO    |DatabaseQuery |   |checking database integrity (may take a while)
2021-07-24 14:58:38.192846|WARNING |Accounting    |   |Unable to open licensekey.dat, falling back to limited functionality
2021-07-24 14:58:38.192979|CRITICAL|Accounting    |   |The default license has expired. Please use the latest server version
```

thanks :)

**Comment by qdm12** [comment]
Oops small bug from my side, it was pushing to tag `:master` instead of `:latest` sorry about that!

- Build was fixed in 02a04443a90c48cc8e754e9d67195005f0c75426 to push to `:latest`
- Added documentation if you get that expired license message in fef7174ea3d3d6bcd966c44460df1c520e5d9f36 [here](https://github.com/qdm12/teamspeak-server-alpine#expired-license) so that even if I get savagely murdered by a bus, you can still update the server version for the next few years without my help 😉

**Comment by pazport** [comment]
> 
> 
> Oops small bug from my side, it was pushing to tag `:master` instead of `:latest` sorry about that!
> 
>     * Build was fixed in [02a0444](https://github.com/qdm12/teamspeak-server-alpine/commit/02a04443a90c48cc8e754e9d67195005f0c75426) to push to `:latest`
> 
>     * Added documentation if you get that expired license message in [fef7174](https://github.com/qdm12/teamspeak-server-alpine/commit/fef7174ea3d3d6bcd966c44460df1c520e5d9f36) [here](https://github.com/qdm12/teamspeak-server-alpine#expired-license) so that even if I get savagely murdered by a bus, you can still update the server version for the next few years without my help 😉

nice 1 thanks got it working.

you need to add ``accept`` on end of ``TS3SERVER_LICENSE`` after that it works mate :) keep up the work. i use this for me and friends when gaming

**Comment by qdm12** [comment]
Happy to help, enjoy! 👍
